source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -421,6 +371,43 @@ This example specifies the following driver behavior in a pool of
    :emphasize-lines: 3-5
    :dedent:
 
+.. _mcs-logger-settings:
+
+Logger Settings
+---------------
+
+Chain the `applyToLoggerSettings() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#applyToLoggerSettings(com.mongodb.Block)>`__
+method to modify the logging behavior of the driver.
+
+The following table describes all the methods you can chain to your
+settings to modify the driver's logger behavior:
```

Comment:
```suggestion
settings to modify the driver's logging behavior:
```

Reply:
I think this is correct, because it is the behavior of the logger. The logger is a specific element of the driver. 

Both could be accurate, but I went with logger for consistency with the section.

Reply:
Had a couple minutes to take a look here and had a couple comments:

Question:
Wouldn't "logging behavior" be more consistent with line 380: "method to modify the logging behavior of the driver"?